### PR TITLE
Assigment 4

### PR DESCRIPTION
**Pull request by YIvanX**

### CODE REVIEW

File: Assignment4/YIvanX-190162/task06.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/YIvanX-190162/task06.py]: won't work in Python. This is notebook magic

File: Assignment4/YIvanX-190162/task06.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+g.add((ns.Smith, VCARD.Family, Literal("Smith")))
+g.add((ns.JaneSmith,VCARD.FN, Literal("Jane Smith")))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+# TO DO
+g.add((ns.UPM,RDF.type,ns.University))
+g.add((ns.JhonSmith,VCARD.Works,ns.UPM))
```
Comment (dgarijo) [inline Assignment4/YIvanX-190162/task06.py]: vcard does not have a Works property. Create your own in a new ns

**Comment by dgarijo** [comment]
Minimal changes required

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Comment by YIvanX** [comment]
changes done

File: Assignment4/YIvanX-190162/task06.py (hunk 2 above)
Comment (YIvanX) [reply Assignment4/YIvanX-190162/task06.py]: resolved

**Pull request merged**